crypto/bio/bss_file.c

```diff
@@ -89,6 +89,10 @@
 
 # if !defined(OPENSSL_NO_STDIO)
 
+#  if defined(OPENSSL_SYS_MSDOS) && !defined(OPENSSL_SYS_WINDOWS)
+#   include <libc/unconst.h>
+static void dosify_filename(const char *filename);
+#  endif
```

Comment:
> why uplink?

Solely because it's already there, so one doesn't need to make up more knobs...

On additional note, why not use the opportunity break out target to separate file in Configuraions. It's not like we plan to regularly perform regression tests on the platform in question, is it?

[For public reference. Last sentence is kind of a echo of internal discussion we had regarding what belongs in 10-main.cf and why would one single out single platform like effectively suggested above. My rationale is that sometimes we don't actually *know* if specific config is working and have to make an assumption. Or rather refuse to make one and simply go with status-quo of not knowing. But once we get to *learn* something about platform that we are not [ever] testing ourselves, learn by talking to people, we shouldn't feel bad about making the said assumption on per-platform basis and denote this fact by moving platform out of 10-main.cf.]